testing/deployer/topology/compile.go

```diff
@@ -342,7 +342,7 @@ func compile(logger hclog.Logger, raw *Config, prev *Topology) (*Topology, error
 				wrk.Node = n
 				wrk.NodeVersion = n.Version
 				if n.IsV2() {
-					wrk.Workload = wrk.ID.Name + "-" + n.PodName()
```

Comment:
bug2: The `Node.PodName()` method creates a unique string for use on the system docker. This has a much longer name due to needing additional unique strings to disambiguate on the system when multiple tests are in flight.

For a resource this was not needed and the only thing needed here is the node name, which gets us back down under the `64` character limit in the common case.